Source/Core/Core/HW/GCPadEmu.h

```diff
@@ -8,7 +8,15 @@
 
 #include "InputCommon/ControllerEmu/ControllerEmu.h"
 
+struct GCPadStatus;
```

Comment:
How does this work? Doesn't it need a full declaration of `GCPadStatus` for the `GetInput()` method?